src/diffusers/pipelines/pipeline_utils.py

```diff
@@ -1021,7 +1025,7 @@ def load_module(name, value):
 
             # 6.2 Define all importable classes
             is_pipeline_module = hasattr(pipelines, library_name)
-            importable_classes = ALL_IMPORTABLE_CLASSES if is_pipeline_module else LOADABLE_CLASSES[library_name]
+            importable_classes = ALL_IMPORTABLE_CLASSES
```

Comment:
This is unnecessary, we can always just use "ALL Importable classes" as long as names differ (which we assume right now)